website/addons/wiki/model.py

```diff
@@ -32,6 +32,10 @@
 
 class AddonWikiNodeSettings(AddonNodeSettingsBase):
 
+    @property
+    def complete(self):
```

Comment:
@samchrisinger, you can actually just set this as a class variable if you want.

```python
class AddonWikiNodeSettings(...):
    complete = True
```

For good measure you may want to add `has_auth` as well.